src/utils.ts

```diff
@@ -198,6 +198,19 @@ export function readFileSync(filePath) {
   return fs.readFileSync(filePath, "utf-8");
 }
 
+export function doesPathExists(inputPath: string) {
+  return pathExists(inputPath);
+}
+export function readJson(jsonPath: string) {
+  return readJSON(jsonPath);
+}
+
+export function writeJson(jsonPath: string, object: any) {
+  return writeJSON(jsonPath, object, {
+    spaces: vscode.workspace.getConfiguration().get("editor.tabSize") || 2,
+  });
+}
+
```

Comment:
Is this wrapper really required ? Since we are not adding any extra functionality here, plus these functions are also not getting called from multiple places.